syntax = "proto3";

import "protocol/heartbeat/heartbeat_packet.proto";
import "protocol/version/version_check_packet.proto";
import "protocol/version/version_download_packet.proto";


service HeartbeatService {
  rpc Heartbeat (HeartbeatRequest) returns (HeartbeatResponse){}
}

service VersionCheckService {
  rpc VersionCheck (VersionCheckRequest) returns (VersionCheckResponse) {}
}

service VersionDownloadService {
  rpc VersionDownload (VersionDownloadRequest) returns (VersionDownloadResponse) {}
}
